python/skupper_router/management/skrouter.json

```diff
@@ -1229,6 +1229,28 @@
                     "graph": true,
                     "description": "The number of bytes sent from servers to clients on all connections to this listener."
                 },
+                "encryptedBytesIn": {
+                    "type": "integer",
+                    "graph": true,
+                    "description": "The number of encrypted bytes sent from clients to servers on all connections to this listener. This will be zero if the tcpListener does not have an associated sslProfile."
```

Comment:
Should be "total encrypted bytes"?  Is it "to servers" or is it "from all clients that attached via this listener"?  

The "to servers on all connections" doesn't make sense to me, but I usually defer to jross on all matters schema related...

Reply:
yep, we can change it to whatever @ssorj comes up with.